src/content/en/2020/seo.md

```diff
@@ -83,493 +83,603 @@ Response Code | All Devices | Mobile | Desktop
 
-When analyzing the usage of the title tag, it was found that 98.75% of desktop homepages are featuring one, while 98.7% of mobile homepages are. This represents a slight improvement vs. [last year](../2019/seo#page-titles) when it was found that 97.1% of mobile pages were featuring a title tag.
+When analyzing the usage of the `title` tag, we found that 99% of desktop and mobile home pages have one. This represents a slight improvement since [last year](../2019/seo#page-titles), when 97% of mobile pages had a `title` tag.
 
-The median homepage features a page title that is six words long. There is no difference in the word count between the mobile and desktop contexts within our dataset. This suggests that the page title element is an element that is not modified between different page template types.
+The median home page features a page title that is six words long. There is no difference in the word count between the mobile and desktop contexts within our dataset. This suggests that the page title element is an element that is not modified between different page template types.
 
 {{ figure_markup(
   image="seo-title-word-count.png",
-  caption="Title word count",
-  description="Bar chart showing the number of words in the title tag per percentile (10, 25, 50, 75, and 90). The median homepage features a page title that is six words long. There is no difference in the word count between the mobile and desktop contexts within our dataset",
+  caption="Distribution of the number of words per page title.",
+  description="Bar chart showing the number of words in the title tag per percentile (10, 25, 50, 75, and 90). The median home page features a page title that is six words long. There is no difference in the word count between the mobile and desktop contexts within our dataset",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=2028212539&format=interactive",
   sheets_gid="775602646",
   sql_file="pages_wpt_bodies_by_device_and_percentile.sql"
-  )
-}}
+) }}
 
-The median page title character count is 38 characters on both mobile and desktop. Interestingly, this is up from 20 characters on desktop and 21 characters on mobile from [last year’s analysis](../2019/seo#page-titles). The disparity between the contexts has disappeared year-over-year except within the 90th percentile wherein there is a one character disparity.
+The median page title character count is 38 characters on both mobile and desktop. Interestingly, this is up from 20 characters on desktop and 21 characters on mobile from [last year's analysis](../2019/seo#page-titles). The disparity between the contexts has disappeared year-over-year except within the 90th percentile wherein there is a one character difference.
 
+{# TODO(analysts): The y-axis label should be "Number of characters". #}
```

Comment:
@Tiggerito

Reply:
301 @antoineeripret

Reply:
@Tiggerito : Done and file added to the commit.